experiment/autobumper/bumper/bumper.go

```diff
@@ -286,6 +287,21 @@ func Run(o *Options) error {
 			}
 		}
 
+		// Check to see if the proper fork exists and if it does not, create one.
+		// TODO(mpherman): Handle case where account has repo of same name as one we are trying to autobump but is not a fork of it (e.g. test-infra)
+		_, err := gc.GetRepo(o.GitHubLogin, o.GitHubRepo)
+		if err != nil {
+			if err.Error() == getRepo404ErrMsg {
+				logrus.Infof("fork needed for autobump does not exist, creating new fork of %s/%s for user %s.", o.GitHubOrg, o.GitHubRepo, o.GitHubLogin)
+				_, err := gc.CreateFork(o.GitHubOrg, o.GitHubRepo)
```

Comment:
nit, you can combine this line with the line below

Reply:
Done